main.js

```diff
@@ -27,6 +37,13 @@ function createElementAttribute(name) {
   };
 }
 
+function createStringElementAttribute(name) {
```

Comment:
I guess we can just ignore those cases :P Although setting the attribute on get isn't quite right. It need to be done on creation. For example, if you say style: 'background: red', you'll want the element to have the red background when it gets created rather than when style is accessed.